itests/org.openhab.core.tests/src/main/java/org/eclipse/smarthome/core/library/types/QuantityTypeTest.java

```diff
@@ -291,6 +292,12 @@ public void testMWh() {
         assertEquals("1000000 Wh", energy.toUnit("Wh").toString());
     }
 
+    @Test
+    public void testMilliseonds() {
```

Comment:
This test doesn't test your changes. Because the conversion is already part of the automatic conversion even without your newly added const this test will succeed. What you want to test if your new constant works. Something like this:
```
    @Test
    public void testMilliSeconds() {
        Quantity<Time> oneSecond = Quantities.getQuantity(new BigDecimal("1"), Units.SECOND);
        Quantity<Time> millisecond = oneSecond.to(SmartHomeUnits.MILLISECOND);
        assertThat(millisecond.getUnit().toString(), is("ms"));
        assertThat(millisecond.getValue().doubleValue(), is(closeTo(1000, DEFAULT_ERROR)));
    }
```
This test maybe is better added in `SmartHomeUnitsTest`